src/main/java/edu/ucsb/cs156/gauchoride/repositories/RideRequestRepository.java

```diff
@@ -0,0 +1,13 @@
+package edu.ucsb.cs156.gauchoride.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import edu.ucsb.cs156.gauchoride.entities.RideRequest;
+
+import java.util.Optional;
+
+@Repository
+public interface RideRequestRepository extends CrudRepository<RideRequest, Long> {
+    Iterable<RideRequest> findAllByUserid(Long userid);
```

Comment:
If you change this to riderUserId (which I suggest that you do), you'll need to change this too.